recipe-client-addon/test/browser/browser_NormandyDriver.js

```diff
@@ -1,24 +1,24 @@
 "use strict";
 
-Cu.import("resource://shield-recipe-client/test/browser/Utils.jsm", this);
+Cu.import("resource://gre/modules/Services.jsm", this);
```

Comment:
I don't see `Services` or `Console` being used here or in the other test? Am I missing something?